sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -937,6 +937,14 @@ object SQLConf {
       .timeConf(TimeUnit.SECONDS)
       .createWithDefault(0L)
 
+  val THRIFTSERVER_FORCE_CANCEL =
+    buildConf("spark.sql.thriftServer.interruptOnCancel")
+      .doc("When true, all running tasks will be interrupted if one cancels a query. " +
+        "When false, all running tasks will be remained until finished.")
+      .version("3.2.0")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
I wasn't aware that cancelling a job group leaves currently running tasks running to completion.
Why not set it true? What are the disadvantages?

Reply:
The reason is same with `Why the default behavior of sc.cancelJob does not interrupt task`, related JIRA [SPARK-17064](https://issues.apache.org/jira/browse/SPARK-17064). And also I think the default value set to false can keep the same behavior with early version.